### PR TITLE
specify license

### PR DESCRIPTION
**Pull request by diclophis**

### CODE REVIEW

**Comment by diclophis** [comment]
should provide details for: https://github.com/mavenlink/alaska/issues/1

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2012 Mavenlink, Inc.
```
Comment (cantino) [inline LICENSE]: 2012?

Comment (diclophis) [inline LICENSE]: https://github.com/mavenlink/brainstem/blob/master/LICENSE ...

I mean, thats when it was written, not sure if it changes what date it applies to the thing its attached too...

IANAL

Comment (cantino) [inline LICENSE]: You can update it to 2015

Comment (diclophis) [inline LICENSE]: ok

richard_stallman_-_fete_de_l humanite_2014_-_010

**Pull request merged**